client/components/site-selector/index.jsx

```diff
@@ -246,6 +246,42 @@ export default React.createClass( {
 		);
 	},
 
+	renderStarredSites() {
+		const sites = this.props.sites.getStarred();
```

Comment:
I don't have a strong preference, they are "sites" in the context of the function.